app/views/shared/components/_outline_primary_button.html.erb

```diff
@@ -1,3 +1,3 @@
-<button class="flex items-center gap-2 rounded px-4 py-2 border border-primary">
```

Comment:
Looks good, can you rename the file to _outline_button_secondary.html.erb

Reply:
Done